### ISSUE TITLE
Repeat missing type parameter documentation

### ISSUE THREAD

**Issue by atifaziz** [morelinq/MoreLINQ#148]
Warning running `builddocs.cmd` on cc6932f74cfcc7476bcc99db10f450bfdcb010c0:

```
BuildAssembler : warning : ShowMissingComponent: [M:MoreLinq.MoreEnumerable.Repeat``1(System.Collections.Generic.IEnumerable{``0},System.Int32)] Missing <typeparam name="T"/> documentation [A:\MoreLINQ\main\docs\api\Working\BuildReferenceTopics.proj]
```

**Comment by VladimirRybalko** [comment]
@atifaziz I've created a request to fix it.

**Comment by atifaziz** [comment]
Closed by 5fb766847f0b931ecadfe4cf7d0d196224e353c6 from PR #182
